lib/Migration/Version2040Date20210908101001.php

```diff
@@ -22,79 +43,79 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 		$schema = $schemaClosure();
 
 		$table = $schema->createTable('calendar_appt_configs');
-		$table->addColumn('id', Types::BIGINT, [
+		$table->addColumn('id', 'bigint', [
```

Comment:
Why don't you use the `Types` constants anymore? You can remove `use OCP\DB\Types;` if you don't use it anymore.

Reply:
was added only for 21